core/templates/dev/head/css/oppia.css

```diff
@@ -870,6 +870,39 @@ textarea {
   border: none;
 }
 
+/*Prevent Exploration Player From collapse On Small Screen*/
+@media(max-width:768px) {
+/* Disabling navbar-collapse */
```

Comment:
Out of interest, why do we not just remove navbar-collapse altogether if we aren't going to use it? (I haven't investigated in detail, I was just wondering.)